src/context/transaction/TransactionState.test.js

```diff
@@ -19,9 +19,34 @@ test("Returns false if input is null.", () => {
   expect(validations.isAlphanumeric(null)).toEqual(false);
 });
 
-// Validate Timestamp is valid ISO-8601 format
-
 // Validate Quantity is a decimal value
+test("Returns true if input is a positive, decimal value", () => {
```

Comment:
Now that there are multiple functions being tested, "Returns true if input..." might be ambiguous when looking at the test failures in terminal. One suggestion could be to include the function name in the test description, or even better use [Jest's describe feature](https://www.valentinog.com/blog/jest/)

Reply:
Let's merge this and then next PR will be adding describe to all test suites.